src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" 
```

Comment:
Please fix the formatting in the whole file to not break lines if it's not necessary:

```suggestion
      <a href="#home" class="logo">
        <img src="images/logo.png" alt="logo">
      </a>
```

Please do that to other elements, it will improve the readability of code :)